.github/workflows/linux.yml

```diff
@@ -36,7 +36,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        ossl-branch: [openssl-3.4.0, master]
+        ossl-branch: [openssl-3.5.0, master]
```

Comment:
Would it make sense to target OpenSSL 3.4.1 instead, to enable testing of the algorithms in oqs-provider that are otherwise only available natively in OpenSSL 3.5.0?

Reply:
Since it's a matrix variable in Github worklows, why not both a 3.4.x and a 3.5.x?